modules/custom/openy_socrates/src/OpenySocratesFacade.php

```diff
@@ -82,33 +82,33 @@ public function __call($name, array $arguments) {
    * @param array $services
    *   Services.
    */
-  public function collectDataServices(array $services) {
+  public function collectDataServices($service, $priority) {
     $todo_services = [];
-    foreach ($services as $priority => $allservices) {
+//    foreach ($services as $priority => $allservices) {
       /*
        * @var integer $key
        * @var OpenyDataServiceInterface $service
        */
-      foreach ($allservices as $key => $service) {
+//      foreach ($allservices as $key => $service) {
         foreach ($service->addDataServices($todo_services) as $method) {
           $this->services[$method][$priority] = $service;
           krsort($this->services[$method]);
         }
       }
-    }
-  }
+//    }
+//  }
 
   /**
    * Setter for services tagged with 'openy_cron_service' tag.
    *
    * @param array $services
    *   Services.
    */
-  public function collectCronServices(array $services) {
+  public function collectCronServices($service, $priority, $periodicity) {
     /** @var OpenyCronServiceInterface $service */
-    foreach ($services as $periodicity => $service) {
+//    foreach ($services as $periodicity => $service) {
```

Comment:
Is this finished PR?
I think we should avoid commenting the old code.

Reply:
@danylevskyi , this PR is just a possible solution to update Migrate module to the latest version.
All other changes are taken from one of the previous PRs, where Andrii has tried to update Drupal to v8.5.